lib/screens/home_screen.dart

```diff
@@ -83,7 +73,7 @@ class _HomeScreenState extends State<HomeScreen>
                   ],
                 )),
             floatingActionButton: FloatingActionButton(
-              onPressed: () => _createTodoForm(context, todoBloc),
+              onPressed: () => _createTodoForm(context, todoBloc, state),
               child: Icon(
                 Icons.add,
                 color: AppColorPalette().primaryColor,
```

Comment:
Why are you not using Theme.of(context) ...

Reply:
Theme.of(context) is the definitely the recommended way. Made the changes.